common/gas_limits.go

```diff
@@ -0,0 +1,7 @@
+package common
+
+const (
+	EVMSend = 21000
```

Comment:
```suggestion
	EVMSendGasLimit = 21000
```

Reply:
The file name is already `gas_limits`, I feel adding the word here is redundant , but don't feel strongly about it